src/main/webapp/resources/js/pages/announcement/AnnouncementsTable.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { forwardRef, useContext, useImperativeHandle } from "react";
```

Comment:
Comments

Reply:
Updated in 3611183a5b981a594baf068381c38cd40427d260